File: src/app/classes/product.ts
export class Product {
  private _productId: number;
  private _productName: string;
  private _productPrice: number;
  private _quantity: number;

  get productId(): number {
    return this._productId;
  }

  set productId(value: number) {
    this._productId = value;
  }

  get productName(): string {
    return this._productName;
  }

  set productName(value: string) {
    this._productName = value;
  }

  get productPrice(): number {
    return this._productPrice;
  }

  set productPrice(value: number) {
    this._productPrice = value;
  }

  get quantity(): number {
    return this._quantity;
  }

  set quantity(value: number) {
    this._quantity = value;
  }
}

File: backend/Ezeeshop-CustomerService/Ezeeshop-OrderServiceProducer/src/main/java/com/ezeeshop/entity/Product.java
package com.ezeeshop.entity;

import java.util.ArrayList;
import java.util.List;


public class Product {

	
	private Long productId;
	private String productName;
	private Long productPrice;
	private int quantity;
	
	private List<Order> orders = new ArrayList<Order>();
	public int getQuantity() {
		return quantity;
	}
	public void setQuantity(int quantity) {
		this.quantity = quantity;
	}
	public Long getProductId() {
		return productId;
	}
	public void setProductId(Long productId) {
		this.productId = productId;
	}
	public String getProductName() {
		return productName;
	}
	public void setProductName(String productName) {
		this.productName = productName;
	}
	public Long getProductPrice() {
		return productPrice;
	}
	public void setProductPrice(Long productPrice) {
		this.productPrice = productPrice;
	}
	public List<Order> getOrders() {
		return orders;
	}
	public void setOrders(List<Order> orders) {
		this.orders = orders;
	}
	
	public void addOrders(List<Order> orders) {
		this.orders = orders;
	}
	
	
}
